src/_resources/fact-sheet-minimum-account-requirements-for-transit-riders.md

```diff
@@ -0,0 +1,112 @@
+---
+date: "2023-08-01T17:00:00-07:00"
+title: "Fact Sheet: Minimum Account Requirements for Transit Riders"
+category: Case studies
+tags:
+  - Contactless payments
```

Comment:
I think this needs to be sentence case `Contactless Payments` for the filtering to work